client/src/test/diagnostics.test.ts

```diff
@@ -199,47 +185,23 @@ describe('Should skip non-amp page', () => {
   const docUri = getDocUri('amp/non-amp.html');
 
   it('Skips NON AMP HTML amp/non-amp.html page', async () => {
-    await testDiagnostics(docUri, [
-      {
-        message: 'Doctype must be declared first.',
```

Comment:
Same as earlier comment.
Why all the "Doctype" errors have been removed?

Reply:
Same as earlier, the latest version validator no longer generates this error.